sqldelight-gradle-plugin/src/test/fixtures/javadoc/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE test (
+  _id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT
+);
+
+/**
+ Javadoc comment yo.
```

Comment:
Javadoc lines usually start with ` * ` and the IDE automatically puts them there with the closing thing when you press enter on the opening of the comment.